platform/config/ConfigService.ts

```diff
@@ -120,6 +120,7 @@ export class ConfigService {
   }
 
   private markAsHandled(path: ConfigPath) {
+    this.log.debug(`Marking config path as handled: ${path}`);
```

Comment:
`this.log` was marked as unused, so I just wanted to actually use it instead of removing it and then adding it back. This felt like a good use of it.